draft-ietf-quic-tls.md

```diff
@@ -850,55 +939,45 @@ if packet_type == Initial:
                      len(token)
 ~~~
 
```

Comment:
Benjamin Saunders pointed out on Slack that the long header pseudo code does not check for out of bounds condition when the packet is small.

Reply:
Yes; as an implementer (and the reason Benjamin asked about this) I looked at the example code more than the surrounding prose, and incorrectly assumed the offset adjustment only applies for short header packets. It would be nice to clarify this somehow, either by duplicating that bit of code for the long header example code, or by separating it more from the short header example code to clarify that it applies to both.